Assets/Scenes/MainMenu.unity

```diff
@@ -947,11 +947,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-<<<<<<< HEAD
```

Comment:
One conflict on MainMenu scene

Reply:
Heads up, I accepted the HEAD in my PR (#12)

Reply:
My comment is wrong on this. I was reading the removal as a conflict